Makefile.core.mk

```diff
@@ -50,11 +54,22 @@ generate: generate-values generate-types vfsgen
 
 clean: clean-values clean-types
 
-default: mesh
+default: mesh controller
 
 mesh: vfsgen
 	go build -o ${GOBIN}/mesh ./cmd/mesh.go
 
+controller: vfsgen
+	go build -o ${CONTROLLER_BUILD}/_output/bin/istio-operator ./cmd/manager
+
+controller-image: build-controller-image push-controller-image
+
+build-controller-image: controller
+	docker build -t istionightly/operator:${VERSION} -f ${CONTROLLER_BUILD}/Dockerfile .
+
+push-controller-image:
+	docker push istionightly/operator:${VERSION}
```

Comment:
please don't use istionightly

Reply:
Should we use `docker.io/istio` instead?

Reply:
I added docker.io/istio as a default.